metric/bigtable/src/main/java/com/spotify/heroic/analytics/bigtable/BigtableAnalyticsModule.java

```diff
@@ -87,13 +91,14 @@ public AnalyticsReporter reporter(final HeroicReporter reporter) {
     public Managed<BigtableConnection> connection(
         final AsyncFramework async, final ExecutorService executorService
     ) {
+        Optional<Integer> batchSize = empty();
```

Comment:
Inline this variable, or declare as a default constant to follow the local convention?